### ISSUE TITLE
Installer changes OS language

### ISSUE THREAD

**Issue by ahousseini** [munki/createOSXinstallPkg#9]
The installer changes the language on the machine. Normal behaviour?

e.g. Upgrade a OS X 10.9 machine in german to OS X 10.11 and the machine is in english after the reboot. OSXInstallerPKG was created on a machine with OS X 10.11 with german as primary language. The installer app was downloaded from the same machine with a german Apple ID.

**Comment by gregneagle** [comment]
It's just running Apple's installer. No language choices are being made.

**Comment by peterkelm** [comment]
Are you sure? Line 728 in createOSXinstallPkg defines the OS default language... to English. Change 'en' to e.g. 'de' and you install a German language OS. In fact, it sets both German as the default language (e.g. for the login screen) AND the German keyboard map.

**Comment by gregneagle** [comment]
Excellent point. I would accept a PR that added a --language flag that allowed one to set that during package creation.

**Comment by peterkelm** [comment]
Agreed. I think I'll look into that over the next weekend(s)... for now I've just change the language code in that respective line to whatever I needed.
